src/dotnet/commands/dotnet-new/CSharp_xunittest/project.json.template

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.0.0-*",
+
+    "dependencies": {
+        "Microsoft.NETCore.App": {
+            "type": "platform",
+            "version": "1.0.0-rc2-23931"
```

Comment:
@blackdwarf See [my PR](https://github.com/dotnet/cli/pull/2331), especially here:  https://github.com/dotnet/cli/blob/rel/1.0.0/src/dotnet/commands/dotnet-new/CSharp_Console/project.json.pretemplate

These templates are now moved to "pretemplate", and you can't hard code the version of Microsoft.NETCore.App.